collect_app/src/main/res/values/strings.xml

```diff
@@ -537,5 +537,18 @@
     <string name="ethiopian_month_hamlie">Hamlie</string>
     <string name="ethiopian_month_nehasie">Nehasie</string>
     <string name="ethiopian_month_pagumien">Pagumien</string>
-    <string name="ethiopian_date">%1$s (%2$s)</string>
+    <string name="custom_date">%1$s (%2$s)</string>
```

Comment:
Can you add a comment above for translators? I'm thinking something like "Displays dates in specialized calendars with the Gregorian dates in parentheses. This is translated so right-to-left languages can specify the correct order." Would be nice if it could be shorter but I think all the info is helpful.